src/views/AboutView.vue

```diff
@@ -3,7 +3,8 @@
     <v-card-title class="justify-center">
       {{ configStore.general.title ?? 'Delft-FEWS Web OC' }}
     </v-card-title>
-    <v-card-text v-if="configStore.activeComponents.length > 0">
+    <v-card-text v-if="configStore.activeComponents.length > 0"> </v-card-text>
+    <v-card-text v-else-if="configStore.activeComponents.length > 1">
       Select one of the following options to get started.
     </v-card-text>
     <v-card-text v-else>
```

Comment:
Empty v-card-text seems weird here, could make it:
```suggestion
    <v-card-text v-else-if="configStore.activeComponents.length > 1">
      Select one of the following options to get started.
    </v-card-text>
    <v-card-text v-else-if="configStore.activeComponents.length === 0">
```

Reply:
if the configStore.activeComponents.length is <= 1 then nothing should be shown, so also if it is === 1